src/cosl/coordinated_workers/worker.py

```diff
@@ -340,6 +349,18 @@ def _on_collect_status(self, e: ops.CollectStatusEvent):
             statuses.append(BlockedStatus("Missing relation to a coordinator charm"))
         elif not self.cluster.relation:
             statuses.append(WaitingStatus("Cluster relation not ready"))
+
+        try:
+            self._charm_tracing_endpoint, self._charm_tracing_cert = self.charm_tracing_config()
+        except CharmTracingTLSConfigurationError:
```

Comment:
Can we avoid the raise-and-swallow by changing the approach underneath?

Reply:
as I said in the description, let's not focus on the implementation just yet